loader.js

```diff
@@ -26,25 +26,23 @@ var Module = null;
      }
 
      function img(src) {
-       var img = new Image();
-       img.src = src;
-       return img;
+       var _img = new Image();
+       _img.src = src;
+       return _img;
      }
 
      // yea, this is a hack
-     if (/archive\.org$/.test(document.location.hostname)) {
-       var images = { ia: img("/images/ialogo.png"),
-                      mame: img("/images/mame.png"),
-                      mess: img("/images/mess.png"),
-                      dosbox: img("/images/dosbox.png")
-                    };
-     } else {
-       images = { ia: img("other_logos/ia-logo-150x150.png"),
-                  mame: img("other_logos/mame.png"),
-                  mess: img("other_logos/mess.png"),
-                  dosbox: img("other_logos/dosbox.png")
-                };
-     }
+     var images = (/archive\.org$/.test(document.location.hostname)) ?
+        { ia: img("/images/ialogo.png"),
+            mame: img("/images/mame.png"),
+            mess: img("/images/mess.png"),
+            dosbox: img("/images/dosbox.png")
+        } : {
```

Comment:
I can't merge that; the indentation is terrible. The colon of the tertiary operator must line up with the question mark.